File: Modul-4/Soal-3/PRAK403-2410817210008-AhmadUlyani.py
def bilangan(a, b):
    hasil = []
    
    if a < b:  # Kasus a lebih kecil dari b
        start, end = a, b
        while start <= b and end >= a:
            hasil.append(f"{start} {end}")
            start += 1
            end -= 1
    else:  # Kasus a lebih besar dari b
        start, end = a, b
        while start >= b and end <= a:
            hasil.append(f"{start} {end}")
            start -= 1
            end += 1

    return " - ".join(hasil)

a, b = map(int, input().split())
print(bilangan(a, b))

File: Modul-4/Soal-3/PRAK403-2410817210008-AhmadUlyani.c
#include <stdio.h>

void bilangan(int a, int b) {
    int start = a, end = b;

    if (a < b) { // Kasus a lebih kecil dari b
        while (start <= b && end >= a) {
            printf("%d %d", start, end);
            if (start < b && end > a) {
                printf(" - ");
            }
            start++;
            end--;
        }
    } else { // Kasus a lebih besar dari b
        while (start >= b && end <= a) {
            printf("%d %d", start, end);
            if (start > b && end < a) {
                printf(" - ");
            }
            start--;
            end++;
        }
    }
    printf("\n");
}

int main() {
    int a, b;
    scanf("%d %d", &a, &b);
    bilangan(a, b);
    return 0;
}